ToolBox/ExanteDownloader/ExanteDataDownloader.cs

```diff
@@ -0,0 +1,19 @@
+namespace QuantConnect.ToolBox.ExanteDownloader
+{
+    public class ExanteDataDownloader
+    {
+        /// <summary>
+        /// Creates a new instance of <see cref="ExanteDataDownloader"/>
+        /// </summary>
+        public ExanteDataDownloader()
+        {
+        }
+
+        /// <summary>
+        /// Downloads data from Exante data source
+        /// </summary>
+        public void Download()
+        {
+        }
+    }
+}
```

Comment:
Please delete if not implementing

Reply:
I implemented the stubs. Do you need any data from Exante? Alas, I don't know. I will delete them if not.

Reply:
Ideal would be to implement them for the open-source community and so others using Exante locally could use them.

Reply:
> Please delete if not implementing

Reminder